infrastructure/scripts/provision/core.sh

```diff
@@ -14,14 +14,9 @@ wait-for-service-account
 
 echo "Deploying Airy Core with the ${AIRY_VERSION} image tag"
 
-if [[ -f "${INFRASTRUCTURE_PATH}"/airy.yaml ]]; then
-    yq eval '.global.appImageTag="'"${AIRY_VERSION}"'"' -i "${INFRASTRUCTURE_PATH}"/airy.yaml
-    helm install core "${INFRASTRUCTURE_PATH}"/helm-chart/ --values "${INFRASTRUCTURE_PATH}"/airy.yaml --set global.ngrokEnabled="${NGROK_ENABLED}" --timeout 1000s > /dev/null 2>&1
-    wget -qnv https://airy-core-binaries.s3.amazonaws.com/"${AIRY_VERSION}"/linux/amd64/airy
-    chmod +x airy
-    mv airy /usr/local/bin/
-    airy init
-    airy config apply --kube-config /etc/rancher/k3s/k3s.yaml --config "${INFRASTRUCTURE_PATH}"/airy.yaml
-else
-    helm install core "${INFRASTRUCTURE_PATH}"/helm-chart/ --set global.appImageTag="${AIRY_VERSION}" --set global.ngrokEnabled="${NGROK_ENABLED}" --timeout 1000s > /dev/null 2>&1
-fi
+wget -qnv https://airy-core-binaries.s3.amazonaws.com/"${AIRY_VERSION}"/linux/amd64/airy
+chmod +x airy
+mv airy /usr/local/bin/
+airy init
+airy create --provider=local
```

Comment:
I thought this will be something like
```
airy create --kubeconfig /etc/rancher/k3s/k3s.yaml
```
And the helm charts are deployed.

I think we agreed on building the first version with the kubeconfig file there and k3s already setup, but I might be wrong.

Reply:
Yes that's correct, the flag is missing from this line. But all this will go away of course after https://github.com/airyhq/airy/issues/1041

Reply:
Thank you :+1: 
Maybe we don't even need the `--provider` flag for now.